Updated hero to optionally take a list or string for rotating images

krowvin opened the pull request:
Fixes #73

# Hero optionally can rotate images
* Updated the docs page to include this
* Tested locally to ensure it was working

# TODO: Would be great to add a cleaner transition between images (i.e. fade-in/out)

willbreitkreutz commented:
Add to the docs that users should web-optimize images prior to loading them, if you have a good online tool, then we could provide a link to that as well

krowvin commented:
Okay to reference https://tinypng.com/ as the *free* online tool?

Any other suggestions to add to the list of tools?

I will also add a link to the MDN docs as to _why_ you should do this
https://developer.mozilla.org/en-US/docs/Learn/Performance/Multimedia#why_optimize_your_multimedia

willbreitkreutz commented:
@krowvin yeah, tinypng should be fine

willbreitkreutz commented on lib/components/hero.jsx:
```diff
@@ -1,27 +1,71 @@
+  const [currentImage, setCurrentImage] = useState(typeof image === "object" ? image[0] : image);
+  const [currentAlt, setCurrentAlt] = useState(typeof alt === "object" ? alt[0] : alt);
+  // Make sure the alt count matches the images if both are provided as lists
+  if (typeof image === "object" && typeof alt === "object") {
```
How about `Array.isArray()`

krowvin commented:
@willbreitkreutz  

I made the changes for `isArray` we discussed. I also added a few other parameters because I realized I was not actually showing the image rotor (i wanted to test it was working in the docs not just the `swt site`)

I also added the reference to tinypng from Adam's point about image sizes. 

Let me know if you see any issues.
